climada/hazard/tc_tracks_synth.py

```diff
@@ -726,37 +748,60 @@ def _apply_decay_coeffs(track, v_rel, p_rel, land_geom, s_rel):
             continue
         if land_sea - sea_land == 1:
             continue
-        p_decay = _calc_decay_ps_value(track, p_landfall, land_sea - 1, s_rel)
-        p_decay = _decay_p_function(p_decay, p_rel[ss_scale_idx][1],
-                                    track.dist_since_lf[sea_land:land_sea].values)
-        # dont applay decay if it would decrease central pressure
-        p_decay[p_decay < 1] = track.central_pressure[sea_land:land_sea][p_decay < 1] / p_landfall
-        track.central_pressure[sea_land:land_sea] = p_landfall * p_decay
+        S = _calc_decay_ps_value(track, p_landfall, land_sea - 1, s_rel)
+        if S <= 1:
+            # central_pressure at start of landfall > env_pres after landfall:
+            # set central_pressure to environmental pressure during whole lf
+            track.central_pressure[sea_land:land_sea] = track.environmental_pressure[sea_land:land_sea]
```

Comment:
If S <= 1, a decay cannot be applied because it would decrease central pressure. Hence in that case central pressure is simply set to environmental pressure.

Reply:
Why can central pressure not decrease?

Reply:
When a TC makes landfall, its strength should decay, not increase, hence it would be unrealistic to increase pressure drop (=decrease pressure). However S<=1 can only happen if central pressure before landfall is already at the level of an environmental pressure, therefore it should simply stay to that level.

Reply:
Could you please provide a citation for the landfall strength decrease? Not that this is controversial, but we should differentiate in the code when assumptions are backed by external sources.

Reply:
A specific paper would be better as websites are not persistent.